deployment/chainloop/values.yaml

```diff
@@ -103,6 +103,10 @@ controlplane:
     # Overrides the image tag whose default is the chart appVersion.
     # tag: latest
 
+  ## @param controlplane.tlsConfig.secret name of a secret containing TLS certificate to be used by the controlplane grpc server.
+  tlsConfig:
+    secret: ""
```

Comment:
It makes sense to me to first support providing the name of a secret which has the keypair inside. But I have a couple of suggestions, let me know what you think.

### Make the `secret` key in fact an object

so we end up with something like 

```
tlsConfig:
  secret:
    name: ""
```

The reason being two fold, a) we can clearly leave the door open to other implementations of this API such as providing the cert inline and b) we can easily extend the `by secret name` mechanism to include other opts such as keyNames and so on(as you suggested in the past)

## Add more context

Since not any secret will work here but just a secret that has two entries `tls.crt` and `tls.key`, I'd make add a comment pointing it out.

Reply:
Done

Reply:
Great, thanks!